security/BasicAuth.php

```diff
@@ -50,9 +50,19 @@ public static function requireLogin($realm, $permissionCode = null, $tryUsingSes
 		$isRunningTests = (class_exists('SapphireTest', false) && SapphireTest::is_running_test());
 		if(!Security::database_is_ready() || (Director::is_cli() && !$isRunningTests)) return true;
 
+                /*
+                 * Enable HTTP Basic authentication workaround for PHP running in CGI mode with Apache
+                 * Depending on server configuration the auth header may be in HTTP_AUTHORIZATION or
+                 * REDIRECT_HTTP_AUTHORIZATION
+                 * 
+                 * The follow rewrite rule must be in the sites .htaccess file to enable this workaround
+                 * RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+                 */
+		$authHeader = isset($_SERVER['HTTP_AUTHORIZATION']) ? 'HTTP_AUTHORIZATION' :
```

Comment:
Instead of getting the header name to later retrieve from $_SERVER should we just get the value here?